### PR TITLE
Enable to set LogLevel by environment variable

### PR DESCRIPTION
**Pull request by wafuwafu13**
Enable to set LogLevel by environment variable(`MACKEREL_LOG_LEVEL`)

### CODE REVIEW

File: cmd/mackerel-container-agent/main.go [hunk 1]
```diff
@@ -15,6 +15,24 @@ var version, revision string
...
+	case "INFO":
+		logging.SetLogLevel(logging.INFO)
+	case "WARNING":
+		logging.SetLogLevel(logging.WARNING)
+	case "ERROR":
+		logging.SetLogLevel(logging.ERROR)
+	case "CRITICAL":
+		logging.SetLogLevel(logging.CRITICAL)
+	default:
+		logging.SetLogLevel(logging.INFO)
```
Comment (wafuwafu13) [inline cmd/mackerel-container-agent/main.go:33]: ref: https://github.com/mackerelio/golib/blob/cf653fec783435f02255b01c5ccdedc51036dac8/logging/logger.go#L20

**Review by heleeen — APPROVED** [review]
LGTM 👍

**Review by ne-sachirou — APPROVED** [review]
❤️ 🗒️

**Review by hayajo — APPROVED** [review]
LGTM
Thank You!

**Pull request merged**